### PR TITLE
Add info to login to a specific cluster

### PR DESCRIPTION
**Pull request by armandgrillet**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-71351
Includes https://jira.d2iq.com/browse/D2IQ-71352

## Checklist
- [X] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [X] Test all commands and procedures, if applicable.
- [X] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/1.2/tutorials/login-cluster/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+![Login GitHub](/ksphere/kommander/1.2/img/login-identity-provider.png)
+
+The user interface provides instructions to attach `kubectl` to the cluster:
+
+![Connection Instructions](/ksphere/kommander/1.2/img/kubectl-connection-instructions.png)
+
+Once you will have followed these instructions, your local `kubectl` will be able to communicate with the cluster.
+Depending on your rights, you will be able to view and edit different kind of api-resources.
+
+## Switch from a cluster to another
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.2/tutorials/login-cluster/index.md:43]:
```suggestion
## Switch from a cluster to another cluster
```

File: pages/ksphere/kommander/1.2/tutorials/login-cluster/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+If you wish to login to another cluster that is part of your Kommander infrastructure, you will have to follow the steps described above with another cluster.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.2/tutorials/login-cluster/index.md:45]:
```suggestion
To log in to another cluster, that is part of your Kommander infrastructure, use the [Access a cluster](*access-a-cluster) procedure for the new cluster.
```

File: pages/ksphere/kommander/1.2/tutorials/login-cluster/index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+Once `kubectl` is attached to this new cluster, you can switch between clusters using [contexts](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#context). Open-source tools such as [`kubectx`](https://github.com/ahmetb/kubectx) are available to make this operation faster.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.2/tutorials/login-cluster/index.md:47]:
```suggestion
Once `kubectl` is on this new cluster, you can switch between clusters using [contexts](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#context). Open-source tools such as [`kubectx`](https://github.com/ahmetb/kubectx) can make this operation faster.
```

File: pages/ksphere/kommander/1.1/tutorials/login-cluster/index.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+The user interface provides instructions to attach `kubectl` to the cluster:
+
+![Connection Instructions](/ksphere/kommander/1.1/img/kubectl-connection-instructions.png)
+
+Once you will have followed these instructions, your local `kubectl` will be able to communicate with the cluster.
+Depending on your rights, you will be able to view and edit different kind of api-resources.
+
+## Switch from a cluster to another
+
+If you wish to login to another cluster that is part of your Kommander infrastructure, you will have to follow the steps described above with another cluster.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/tutorials/login-cluster/index.md:45]:
```suggestion
To log in to another cluster, that is part of your Kommander infrastructure, use the [Access a Cluster](#access_a_cluster) procedure for the new cluster.
```

File: pages/ksphere/kommander/1.1/tutorials/login-cluster/index.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+Once `kubectl` is attached to this new cluster, you can switch between clusters using [contexts](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#context). Open-source tools such as [`kubectx`](https://github.com/ahmetb/kubectx) are available to make this operation faster.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/tutorials/login-cluster/index.md:47]:
```suggestion
Once `kubectl` is on this new cluster, you can switch between clusters using [contexts](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#context). Open-source tools such as [`kubectx`](https://github.com/ahmetb/kubectx) can make this operation faster.
```

File: pages/ksphere/kommander/1.1/tutorials/login-cluster/index.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+### Messages when attached to a cluster that you do not have access to
+
+The instructions described above actually work for any clusters (management and attached ones) even if you do not have access to it.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/tutorials/login-cluster/index.md:51]:
```suggestion
The [Access a Cluster](#access_a_cluster) instructions work for any clusters (management and attached ones) even if you do not have access to them.
```

File: pages/ksphere/kommander/1.1/tutorials/login-cluster/index.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+Once you will have followed these instructions, your local `kubectl` will be able to communicate with the cluster.
+Depending on your rights, you will be able to view and edit different kind of api-resources.
+
+## Switch from a cluster to another
+
+If you wish to login to another cluster that is part of your Kommander infrastructure, you will have to follow the steps described above with another cluster.
+
+Once `kubectl` is attached to this new cluster, you can switch between clusters using [contexts](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#context). Open-source tools such as [`kubectx`](https://github.com/ahmetb/kubectx) are available to make this operation faster.
+
+### Messages when attached to a cluster that you do not have access to
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/tutorials/login-cluster/index.md:49]:
```suggestion
### Messages when attached to a cluster that you do not have access
```

File: pages/ksphere/kommander/1.1/tutorials/login-cluster/index.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+The instructions described above actually work for any clusters (management and attached ones) even if you do not have access to it.
+
+The difference will be that, once you have attached the cluster, none of the `kubectl` commands will succeed as the user does not have access:
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/tutorials/login-cluster/index.md:53]:
```suggestion
The difference is that, once you have attached the cluster, none of the `kubectl` commands will succeed as the user does not have access:
```

**Review by russurquhart1 — APPROVED** [review]
I gave comments for 1.1, can you please carry those. same changes. over to 1.2 in this repo? THX!!

File: pages/ksphere/kommander/1.1/tutorials/login-cluster/index.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+![Konvoy Landing Page](/ksphere/kommander/1.1/img/konvoy-landing-page.png)
+
+2. Select the cluster you want to log into.
+
+![Generate Kubernetes Token](/ksphere/kommander/1.1/img/generate-kubernetes-token.png)
+
+3. As a user having access to the management cluster, select the `Main` cluster and log in using the identity provider available (GitHub).
+
+![Login GitHub](/ksphere/kommander/1.1/img/login-identity-provider.png)
```
Comment (tillt) [inline pages/ksphere/kommander/1.1/tutorials/login-cluster/index.md:34]: Why is this specific to GitHub?

File: pages/ksphere/kommander/1.2/tutorials/login-cluster/index.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+![Konvoy Landing Page](/ksphere/kommander/1.2/img/konvoy-landing-page.png)
+
+2. Select the cluster you want to log into.
+
+![Generate Kubernetes Token](/ksphere/kommander/1.2/img/generate-kubernetes-token.png)
+
+3. As a user having access to the management cluster, select the `Main` cluster and log in using the identity provider available (GitHub).
+
+![Login GitHub](/ksphere/kommander/1.2/img/login-identity-provider.png)
```
Comment (tillt) [inline pages/ksphere/kommander/1.2/tutorials/login-cluster/index.md:34]: See above.

**Review by tillt — APPROVED** [review]

**Pull request merged**